website/content/en/docs/deployment/cognito/guide.md

```diff
@@ -1,6 +1,6 @@
 +++
-title = "Manual Deployment Guide"
-description = "Deploying Kubeflow with AWS Cognito as identity provider"
+title = "Manifest Deployment Guide"
```

Comment:
Manifest Deployment Guide should have automated and Manual deployment guides as subsections
@ryansteakley

Reply:
@surajkota  does this refer to having the guide-automated.md instead live inside of guide.md?

Reply:
No, like:
```
- Manifest Deployment Guide
--- Automated Deployment Guide
--- Manual deployment Guide
- Terraform Deployment Guide
```
The way it is under Cognito in 1.5 release, it should be under Cognito -> Manifest Deployment Guide
![Screenshot 2022-09-20 at 6 24 57 AM](https://user-images.githubusercontent.com/22246703/191269621-72cf1941-fec7-4c69-bb18-74b38ce63d93.png)

Reply:
https://github.com/awslabs/kubeflow-manifests/pull/403